packages/rrweb-snapshot/src/snapshot.ts

```diff
@@ -242,32 +242,35 @@ export function transformAttribute(
   maskAllText: boolean,
   maskTextFn: MaskTextFn | undefined,
 ): string {
+  if (!value) {
```

Comment:
l: should we change `value`'s type to `string | undefined` in the function signature to reflect that it apparently can be undefined?

Reply:
makes sense!

Reply:
hmm, it seems there is actually a longer tail to this (as a lot of types seem to depend on the return value of this being `string`.)

So IMHO it's correct, but let's look at this separately maybe then!